en/snap/with-plugins.md

```diff
@@ -56,269 +57,309 @@ Here is the list of Content Management System plugins and extensions that are su
    1. Download the plugin from [Zip](https://github.com/veritrans/SNAP-Woocommerce/archive/master.zip).
    2. Extract the plugin, then rename the modules folder as **midtrans-woocommerce**.
    3. Upload the unzipped plugin folder to your WordPress installation's `./wp-content/plugins/` directory.
-   4. Install and activate the plugin from plugins menu within the WordPress admin panel
+   4. **Install and activate** the plugin from plugins menu within the WordPress administration panel.
    5. Proceed to Configuration Process.
-   
-#### WooCommerce Plugin Configuration Process
-1. Go to **WooCommerce -> Settings -> Payments -> Midtrans** menu, fill the configuration fields.
-    - Fill **Title** with text button that you want to display to customer
-    - Select **Environment**. `Sandbox` for testing transaction and `Production` for real transaction
-    - Fill **Merchant ID, Client Key, and Server key**. You can find [this credential on Midtrans MAP Dashboard](/en/snap/preparation.md).
-    - Other configuration are optional, you may leave it as is.
+
+#### Configuring Midtrans WooCommerce Plugin
+To configure WooCommerce plugin, follow the steps given below.
+1. Go to **WooCommerce > Settings > Payments > Midtrans** menu.
+2. Enter **Title**. This text appears on the button displayed to the customer.
+3. Select **Environment** from dropdown. `Sandbox` for testing transaction and `Production` for real transaction.
+4. Enter **Merchant ID**.
+5. Enter **Client Key**.
+6. Enter **Server key**.
+   For more details, refer to [Preparation](/en/snap/preparation.md).
+
+?>***Note***: Other fields are optional.
 
 ![WooCommerece Install](./../../asset/image/WooCommerce-install.gif)
 
-#### WooCommerce Plugin Notification Configuration
-Login to your [Midtrans Account](https://account.midtrans.com/login), select your environment (sandbox/production), go to menu `settings` **->** `configuration` and `settings` **->** `Snap Preference` **->** `System Settings`
 
-| URL Role | Redirect URL|
-|----------|-------------|
-| Payment Notification URL | [your-site-url]/?wc-api=WC_Gateway_Midtrans |
-| Finish Redirect URL | [your-site-url]/?wc-api=WC_Gateway_Midtrans |
-| Error Redirect URL | [your-site-url]/?wc-api=WC_Gateway_Midtrans |
-| Unfinish Redirect URL | [your-site-url]/?wc-api=WC_Gateway_Midtrans |
+#### Configuring WooCommerce Plugin Notification
+To configure the WooCommerce Plugin Notification URL, follow the steps given below.
+1. Login to your MAP account.
+2. Select the **Environment** from drop-down list.
+3. On the Home page, go to **SETTINGS > CONFIGURATION**.
+   *Configuration* page is displayed.
+4. Enter **Payment Notification URL**.
+5. Enter **Finish Redirect URL**.
+6. Enter **Error Redirect URL**.
+7. Enter **Unfinish Redirect URL**.
+8. Click **Update**.
+   A confirmation message is displayed.
+   ![Snap API](./../../asset/image/snap-api-notification-configuration-1.png)
```

Comment:
I saw lots of broken images throughout this page:

> <img width="714" alt="Screen Shot 2020-11-16 at 16 32 19" src="https://user-images.githubusercontent.com/13027142/99236107-57fe8400-2829-11eb-9081-20f5ef64e53b.png">

The file `snap-api-notification-configuration-1.png` is missing.
What is this image about? screenshot of MAP notification url config pages? I think the instruction is clear enough, we don't need the image. Also because currently the pages is already have many images.

Reply:
I will remove the images